.travis.yml

```diff
@@ -1,15 +1,15 @@
-sudo: false
 dist: trusty
 
 language: go
 go_import_path: gopkg.in/src-d/enry.v1
 
 go:
-  - '1.11.x'
-  - '1.10.x'
+  - '1.11.6'
+  - '1.12.1'
```

Comment:
Are we using 1.12.1 instead of 1.12.3 for some specific reason, and if so can you please add a `# brief comment` with a link to the (or an) issue?

Reply:
👍 yes, ok, will move it from PR description here as well